app/models/user.rb

```diff
@@ -4,6 +4,9 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
+  VALID_EMAIL_REGEX = /\A([^@\s]+)@counterworks.jp/
```

Comment:
定数使う場合は書き換えられないように `.freeze` した方が良い
```suggestion
  VALID_EMAIL_REGEX = /\A([^@\s]+)@counterworks.jp/.freeze
```

Reply:
↓でfreezeを使用して修正しました。
e2d3519b45a252256f77554679a0c026278668b3